### PR TITLE
feat: add echo command

### LINKED ISSUE #19: support echo commnd
### Which kiwi functionalities are relevant/related to the feature request?

_No response_

### Which PikiwiDB functionalities are relevant/related to the feature request?

other

### Description

support echo, auth commnd

### Proposed solution

echo: refer to https://redis.io/docs/latest/commands/?name=echo

### Alternatives considered

nothing

### Proposed solution

_No response_

### Alternatives considered

_No response_

### PR DESCRIPTION
**Pull request by Antarctica000**
This PR adds the functionality for the `echo` command to the `add_echo_cmd` branch. The command will return the arguments provided by the user as output.

fix #19 

## Summary by CodeRabbit

- **New Features**
	- Introduced a new command `Echo` that allows users to concatenate arguments and return a formatted response.
	- Added a new constant `kCmdNameEcho` to enhance command name definitions.
  
- **Bug Fixes**
	- No bug fixes were made in this release.

- **Documentation**
	- Updated command table to include the new `Echo` command that requires two parameters.

### CODE REVIEW

**Comment by luky116** [comment]
cmake-build 文件不要传上来

File: src/net/callback_function.h [hunk 1]
```diff
@@ -35,8 +35,7 @@ concept HasSetFdFunction = requires(T t, uint64_t id, int8_t index) {
   { (*t).GetConnId() } -> std::same_as<uint64_t>;        // GetFd return type is int
   { (*t).SetThreadIndex(index) } -> std::same_as<void>;  // SetThreadIndex return type is void
   { (*t).GetThreadIndex() } -> std::same_as<int8_t>;     // GetThreadIndex return type is int8_t
-}
-|| std::is_class_v<T>;  // If T is an ordinary class, the member function is called directly
```
Comment (luky116) [inline src/net/callback_function.h]: 这种没改动的，不要提交上来

**Comment by AlexStocks** [comment]
有文件冲突了，烦请处理下

File: src/cmd_admin.cc [hunk 2]
```diff
@@ -160,6 +160,26 @@ bool PingCmd::DoInitial(PClient* client) { return true; }
...
+    return;
+  }
+
+  std::string response;
+  for (size_t i = 1; i < argc; i++) {
+    if (i > 1) response += " ";
+    response += client->argv_[i];
+  }
+
+  client->SetRes(CmdRes::kEcho, response);
```
Comment (lqxhub) [inline src/cmd_admin.cc]: 这里没必要使用 `client->SetRes` 返回数据,同理,上面也没必要在 `CmdRet` 中 新加一个类型,直接返回对应的数据就行

File: src/cmd_table_manager.cc [hunk 3]
```diff
@@ -56,6 +56,7 @@ void CmdTableManager::InitCmdTable() {
   ADD_SUBCOMMAND(Config, Get, -3);
   ADD_SUBCOMMAND(Config, Set, -4);
   ADD_COMMAND(Ping, 0);
+  ADD_COMMAND(Echo, 0);
```
Comment (lqxhub) [inline src/cmd_table_manager.cc]: 这里应该是 `-1` 具体可以参考redis

File: src/client.cc [hunk 4]
```diff
@@ -62,6 +62,11 @@ void CmdRes::SetRes(CmdRes::CmdRet _ret, const std::string& content) {
     case kPong:
       SetLineString("+PONG");
       break;
+    case kEcho:
+      AppendStringRaw("+");
+      AppendStringRaw(content);
+      AppendStringRaw(CRLF);
```
Comment (loveyacper) [inline src/client.cc]: 调用 AppendSimpleString

**Comment by marsevilspirit** [comment]
I will carry out the subsequent work.

**Review by AlexStocks — APPROVED** [review]

**Review by luky116 — APPROVED** [review]

**Pull request merged**